src/wallet/wallet2.cpp

```diff
@@ -2546,10 +2546,6 @@ void wallet2::process_unconfirmed(const crypto::hash &txid, const cryptonote::tr
   if(unconf_it != m_unconfirmed_txs.end()) {
     if (store_tx_info()) {
       try {
-        // TODO(doyle): ONS introduces tx type stake, we can use this to quickly determine if a transaction is staking
-        // transaction without having to parse tx_extra.
-        bool stake = service_nodes::tx_get_staking_components(tx, nullptr /*stake*/);
-        wallet::pay_type pay_type = stake ? wallet::pay_type::stake : wallet::pay_type::out;
```

Comment:
Could not see where this `pay_type` variable was used. Was getting unused variable warnings also

Reply:
Well, you know, *relatively*, an unused random variable on 1 line of a 15000-line monstrosity is only 0.0067%, so it's 99.9933% pure.